src/main/java/uk/ac/cam/cl/dtg/isaac/dao/EventBookingPersistenceManager.java

```diff
@@ -255,7 +264,11 @@ private EventBookingDTO convertToDTO(final EventBooking eb, final IsaacEventPage
         try {
             UserSummaryWithEmailAddressDTO user = userManager.convertToDetailedUserSummaryObject(userManager.getUserDTOById(eb
                     .getUserId()), UserSummaryWithEmailAddressDTO.class);
-
+            Long reservedById = eb.getReservedBy();
+            if (reservedById != null && !reservedById.equals(0L)) {
+                UserSummaryWithEmailAddressDTO reservingUser = userManager.convertToDetailedUserSummaryObject(userManager.getUserDTOById(reservedById), UserSummaryWithEmailAddressDTO.class);
```

Comment:
I think this is dangerous, and I'd rather the DTO did not contain this information unless you add the requisite permissions framework to check wether the current user is allowed to see the reserving user's PII!

Reply:
Just the user ID is enough in the DTO!

Reply:
Fixed in a2c97014